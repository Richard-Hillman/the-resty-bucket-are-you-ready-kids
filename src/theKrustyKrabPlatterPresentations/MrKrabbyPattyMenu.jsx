/* eslint-disable max-len */
import React from 'react';
import PropTypes from 'prop-types';

const MrKrabbyPattyMenu = () => (
 
  // eslint-disable-next-line max-len
  <div style={{ backgroundColor: '', width: '', height: '', color: '' }}>
   
    {/* ------------------------ */}
   
    <form>
      <div>

      </div>

      <div>
            
      </div>

      <div>
            
      </div>
    
    </form>

    {/* ------------------------ */}

  </div>
);

MrKrabbyPattyMenu.propTypes = {

};

export default MrKrabbyPattyMenu;
