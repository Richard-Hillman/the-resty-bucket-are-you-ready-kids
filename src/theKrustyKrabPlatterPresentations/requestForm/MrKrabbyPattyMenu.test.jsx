import React from 'react';
import { render, cleanup } from '@testing-library/react';
import MrKrabbyPattyMenu from './MrKrabbyPattyMenu';

describe('MrKrabbyPattyMenu component', () => {
  afterEach(() => cleanup());
  it('renders MrKrabbyPattyMenu', () => {
    const { asFragment } = render(<MrKrabbyPattyMenu
      url=""
      method=""
      json=""
      onSubmit={jest.fn()}
      onChange={jest.fn()}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
