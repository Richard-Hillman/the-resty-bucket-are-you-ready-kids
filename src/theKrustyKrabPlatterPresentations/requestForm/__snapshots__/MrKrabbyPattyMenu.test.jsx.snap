// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MrKrabbyPattyMenu component renders MrKrabbyPattyMenu 1`] = `
<DocumentFragment>
  <div
    class="MrKrabbyPattyMenu"
    style="background-color: grey; width: 90vw; height: 90vw; color: black;"
  >
    <form>
      <label
        class="url"
      >
        <input
          name="url"
          placeholder="Type Api Url Here"
          type="text"
          value=""
        />
      </label>
      <br />
      <br />
      <section
        class="formRadio"
      >
        <label
          for="GET"
        >
          GET
        </label>
        <input
          id="get"
          name="method"
          type="radio"
          value="GET"
        />
        <label
          for="POST"
        >
          POST
        </label>
        <input
          id="post"
          name="method"
          type="radio"
          value="POST"
        />
        <label
          for="POST"
        >
          POST
        </label>
        <input
          id="put"
          name="method"
          type="radio"
          value="PUT"
        />
        <label
          for="PATCH"
        >
          PATCH
        </label>
        <input
          id="patch"
          name="method"
          type="radio"
          value="PATCH"
        />
        <label
          for="DELETE"
        >
          DELETE
        </label>
        <input
          id="delete"
          name="method"
          type="radio"
          value="DELETE"
        />
      </section>
      <br />
      <br />
      <label>
        <textarea
          name="json"
        />
        <br />
        <br />
        <button>
          SEND
        </button>
      </label>
    </form>
  </div>
</DocumentFragment>
`;
